demo/billing/mzcompose.yml

```diff
@@ -7,167 +7,48 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
-x-port-mappings:
-  - &grafana 3000:3000
-  - &kafka ${KAFKA_PORT:-9092:9092}
-  - &materialized ${MZ_PORT:-6875:6875}
-  - &schema-registry ${SR_PORT:-8081:8081}
-
 version: '3.7'
+
 services:
   materialized:
     mzbuild: materialized
     volumes:
-      - db-data:/share/billing-demo/data
-    ports:
-     - *materialized
-    command: --workers ${MZ_WORKERS:-1} --differential-idle-merge-effort=1000 --disable-telemetry
+    - db-data:/share/billing-demo/data
+    ports: [6875]
+    command: --disable-telemetry
     environment:
-      - MZ_LOG_FILTER=dataflow=error,info
-      - MZ_DEV=1
+    - MZ_LOG_FILTER=dataflow=error,info
```

Comment:
the error log level for dataflow seems odd, and i feel like we could just get rid of it?

Reply:
Done.